### ISSUE TITLE
Training PN++

### ISSUE THREAD

**Issue by saidharb** [ChrisWu1997/DeepCAD#35]
Hi Chris,

I really like your work and currently I am trying to recreate your metrics for CAD-model reconstruction from point clouds. I built my own training script for PN++, however I cannot achieve the metrics indicated in the supplementary document in your paper. 

After scrutinizing ```pc2cad_train.py``` I saw that you train on point clouds with normals, however in the ```pc2cad.py``` script no normals are used. Therefore my 3 questions:

- Was ```pc2cad_train.py``` or ```pc2cad.py``` used for PN++ training?
- Did you use normals to train PN++? If so how were the normals obtained, because the script json2pc.py only creates point clouds with cartesian coordinates and no normals.
- What was your final validation MSE loss after 200 epochs of training? I reached approximately 0.0705, but my ACC_cmd is only at 74.82 (yours at 85.95) and ACC_param at 66.80 (yours 74.22). The provided pre-trained DeepCAD model cannot be the bottleneck, as I tested it and was able to recreate the metrics in the paper.

Thank you in advance for your effort.

Best regards
Said
